envoimoinscher.php

```diff
@@ -2811,7 +3041,7 @@ public function makeApiCall($params)
 		$offers_quotation = array();
 		if (count($offers_db) == 1) // correction needed for quotation
 		{
-			$cot_cl->getQuotation($quot_info);
+			@$cot_cl->getQuotation($quot_info);
```

Comment:
The problem is that we cannot catch warnings with a try catch, the only way i found is to use set_error_handler or to set warnings as errors, but this is not something we are supposed to change. Maybe there is something i missed?
The warnings you saw was due to a problem with the library when it receive a malformed response (like a 404 or 500 error) so this is something we will fix in the library and then add to the module. After this we will remove the @.